### DISCUSSION TITLE
[Enhancement]: Font switch suggestion: Inter → Pretendard

### DISCUSSION THREAD

**Discussion by matte-black** [ciderapp/Cider#619] [Added Features]
### The feature you would like

The app's design orientation seems to be very similar to Apple's UI design, so it seems to have used [Inter](https://rsms.me/inter/) font similar to [Apple's San Francisco font](https://developer.apple.com/fonts/) throughout the app.

[Inter](https://rsms.me/inter/) is a great font, but some stroke elements are slightly different from [Apple's SF font](https://developer.apple.com/fonts/). A Korean font designer thought the same, and he created [Pretendard](https://cactus.tistory.com/306) to the extent that the open source allows.

![화면 캡처 2022-03-21 175016](https://user-images.githubusercontent.com/102019998/159229571-69c47bc1-c206-40d8-a816-14c07309affe.png)

The English glyphs have refined to fit Apple's design philosophy based on [Inter](https://rsms.me/inter/)(1:1 perfect same to Apple SF in my opinion), and since the font developer is Korean, the Korean glyph is also precisely tailored to match other glyphs based on [Noto Sans CJK KR](https://fonts.google.com/noto/specimen/Noto+Sans+KR). It also supports the [Variable font standard](https://web.dev/variable-fonts/), the next-generation font standard.

![화면 캡처 2022-03-21 175128](https://user-images.githubusercontent.com/102019998/159229623-500245e8-8a2e-416d-aa36-f6c198056f72.png)

With this font, the overall design of the app will not only be more visually integrated with Apple's design language, but also gets support for Korean fonts as an added bonus. Applying this suggestion will greatly improve the overall visual satisfaction of your app. 

 - Pretendard Official Webpage:
https://cactus.tistory.com/306
(written in Korean, so you may need a translator)
 - Pretendard GitHub:
 https://github.com/orioncactus/pretendard
 (written in Korean, so you may need a translator)

### Alternatives and Other Information

 - Pretendard Official Webpage:
https://cactus.tistory.com/306
(written in Korean, so you may need a translator)
 - Pretendard GitHub:
 https://github.com/orioncactus/pretendard
 (written in Korean, so you may need a translator)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maikirakiwi** [comment]
We should give this a test run sometime

**Comment by Amaru8** [comment]
**Hi @matte-black, I have some great news!** This feature is now implemented in [this commit](https://github.com/ciderapp/Cider/commit/2b2dcc809693676a236aa205c46b319fa8c3b038) for the latest development version of **Cider**, and will be included in the next Microsoft Store update.
